apps/design-system/src/utils/connectors/github-connector.tsx

```diff
@@ -0,0 +1,66 @@
+import { IFormDefinition } from '@harnessio/forms'
+import { IInputConfigWithConfigInterface, InputConfigType, RadialOption } from '@harnessio/ui/views'
+
+import {
+  getCloningContainer,
+  getConnectionContainer,
+  getMetadataContainer,
+  getResourcesContainer
+} from './common-connector-utils'
+
+export const GITHUB_CONNECTOR_CATEOGRY = 'Code Repository'
+
+const inputs: IInputConfigWithConfigInterface[] = [
+  {
+    inputType: 'radio',
+    path: 'githubType',
+    label: 'GitHub Type',
+    inputConfig: {
+      inputType: 'radio',
+      options: [
+        { label: 'Cloud', value: 'Cloud', description: 'Connect to Github Cloud', id: 'cloud', title: 'Cloud' },
+        {
+          label: 'Enterprise',
+          value: 'Enterprise',
+          description: 'Connect to Github Enterprise',
+          id: 'enterprise',
+          title: 'Enterprise'
+        }
+      ] as RadialOption[]
```

Comment:
Have we finalized this type name `RadialOption`? Is it supposed to be `RadioOption` instead?

Reply:
Also, why is the typecast needed here?

Reply:
will fix that